src/window/win.cc

```diff
@@ -1784,11 +1827,22 @@ namespace SSC {
         break;
       }
 
-      case WM_COMMAND: {
-        if (w == nullptr) {
-          break;
+      case WM_APP + 1: {
```

Comment:
what is `WM_APP + 1`? probably should be a constant to prevent a collision in the future

Reply:
agreed, i'll make it a constant